atlite/cutout.py

```diff
@@ -260,6 +267,7 @@ def meta(self):
 
     @property
     def shape(self):
+        """Geographical shape of the cutout (size y, size x)."""
```

Comment:
size y and size x are the number of coordinates in either latitudinal or longitudinal direction, not the "shape" but rather something like the size or extent.

Reply:
but this is how the shape of an array is defined: (length of first axis, length of the second axis) and this is also what the cutout.shape is giving you (`len(self.coords["y"]), len(self.coords["x"])`)

Reply:
Ok, so is it then more precise to say shape rather than geographical shape? When reading I was only thinking about a e.g. geopandas shape.

Reply:
You are right, that is ambiguous. the problem is that cutout.shape does  not take the time axis into account, so its not the real shape the underlying data. Perhaps something like "Shape of spatial dimensions (y, x) of the cutout data." would be better?

Reply:
Yes.